docs/BUILD.md

```diff
@@ -1,6 +1,36 @@
 # Building s2n-tls
 
-s2n-tls can be built as follows:
+To use s2n-tls, you must build the library from the source and then include it in your program.
```

Comment:
It doesn't feel correct to say that you *must* build s2n-tls from source to use it. We don't know how customers are consuming s2n-tls-- they could already have a binary. I don't think this wording change is an improvement.